libvips/iofuncs/util.c

```diff
@@ -1774,9 +1774,7 @@ vips_enum_from_nick(const char *domain, GType type, const char *nick)
 	if ((enum_value = g_enum_get_value_by_nick(genum, nick)))
 		return enum_value->value;
 
-	/* -1 since we always have a "last" member.
-	 */
-	for (i = 0; i < genum->n_values - 1; i++) {
+	for (i = 0; i < genum->n_values; i++) {
```

Comment:
Does this change mean that the "last" string literal will no longer accepted by the `vips_enum_from_nick` function? (There are a couple of places in sharp that rely on this behaviour, which I'm happy to remove if so as it's probably not a good thing to rely on anyway.)